### DISCUSSION TITLE
help

### DISCUSSION THREAD

**Discussion by fatihdurmaz** [filamentphp/filament#6186] [Help]
my users column         
'ad',
        'soyad',
        'telefon',
        'rol',
        'email',
        'password',
I change my user table in User model and php artisan make:filament-user this command later error.

**Comment by kcanyildirim** [comment]
make:filament-user adds the code as name, email, password to the db. You can translate it to English and solve the problem.

**Reply by fatihdurmaz** [reply]
Merhaba, benim user tablomda as, soyad gibi ekstra alanlar var. bu alanlar filament de olmadığı için hata veriyor. ekstra alanları migration içerisinde tanımladım ama filament ile yeni kullanıcı oluştururken bunları nasıl gösterebilirim

**Reply by kcanyildirim** [reply]
MakeUserCommand içerisinde getUserData fonksiyonunun altını düzenleyebilirsiniz yada create_users_table migrationunda kullanıcıyı insertleyin. 

```
 Schema::create('users', function (Blueprint $table) {
     xxx
});

        DB::table('users')->insert([
            'name' => 'xx'
             ...
        ]);
```

**Reply by danharrin** [reply]
You don't even need to use our command, it just creates a user to get you started.
